setup.py

```diff
@@ -13,24 +13,34 @@
 with open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
-define_macros = [('HB_NO_MT', '1')]
+define_macros = []
 linetrace = False
 if int(os.environ.get('CYTHON_LINETRACE', '0')):
     linetrace = True
     define_macros.append(('CYTHON_TRACE_NOGIL', '1'))
 
 extra_compile_args = []
-if platform.system() != 'Windows':
-    extra_compile_args.append('-std=c++11')
 
-extension = Extension(
-    'uharfbuzz._harfbuzz',
-    define_macros=define_macros,
-    include_dirs=['harfbuzz/src'],
-    sources=['src/uharfbuzz/_harfbuzz.pyx', 'harfbuzz/src/harfbuzz.cc'],
-    language='c++',
-    extra_compile_args=extra_compile_args,
-)
+if not os.environ.get('USE_SYSTEM_HARFBUZZ'):
+    if platform.system() != 'Windows':
+        extra_compile_args.append('-std=c++11')
+    extension = Extension(
+        'uharfbuzz._harfbuzz',
+        define_macros=[('HB_NO_MT', '1')] + define_macros,
+        include_dirs=['harfbuzz/src'],
+        sources=['src/uharfbuzz/_harfbuzz.pyx', 'harfbuzz/src/harfbuzz.cc'],
+        language='c++',
+        extra_compile_args=extra_compile_args,
+    )
+else:
+    extension = Extension(
+        'uharfbuzz._harfbuzz',
+        define_macros=define_macros,
+        include_dirs=['/usr/include/harfbuzz'],
```

Comment:
probably not cool as https://twitter.com/pcwalton/status/1272293882596700160 maybe I should ask `pkg-config`? not sure about the idiomatic here

Reply:
That wouldn’t work on any system with different HarfBiuzz installation path, and certainly wouldn’t help #45 case where HarfBuzz is most likely not installed there.

Reply:
> where HarfBuzz is most likely not installed in this pat

oh, so let's close this till we get some idea how best we can solve that, thanks